develop/smart-contracts/json-rpc-apis.md

```diff
@@ -0,0 +1,397 @@
+            }],
+            "id":1
+        }'
+        ```
+
+        Ensure to replace the `INSERT_RECIPIENT_ADDRESS` and `INSERT_ENCODED_CALL` with the proper values.
+
+### Gas and Fees
+
+- [`eth_gasPrice`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gasprice){target=\_blank} - returns the current gas price in wei
+
+    ???- "Query Example" 
+
+        ```bash title="eth_gasPrice"
+        curl -X POST https://westend-asset-hub-eth-rpc.polkadot.io \
+        -H "Content-Type: application/json" \
+        --data '{
+            "jsonrpc":"2.0",
+            "method":"eth_gasPrice",
+            "params":[],
+            "id":1
+        }'
+        ```
+
+- [`eth_maxPriorityFeePerGas`](){target=\_blank} - returns the current maxPriorityFeePerGas in wei
+
+    ???- "Query Example" 
+
+        ```bash title="eth_maxPriorityFeePerGas"
+        curl -X POST https://westend-asset-hub-eth-rpc.polkadot.io \
+        -H "Content-Type: application/json" \
+        --data '{
+            "jsonrpc":"2.0",
+            "method":"eth_maxPriorityFeePerGas",
+            "params":[],
+            "id":1
+        }'
+        ```
+
+### State and Storage
+
+- [`eth_getCode`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getcode){target=\_blank} - returns the code at a given address
+
+    - Parameters
+      - Address - contract or account address to query code
+      - Block Value (optional) - quantity or tag of the [block value](#block-parameter-values) to be fetched
+
+    ???- "Query Example" 
+
+        ```bash title "eth_getCode"
+        curl -X POST https://westend-asset-hub-eth-rpc.polkadot.io \
+        -H "Content-Type: application/json" \
+        --data '{
+            "jsonrpc":"2.0",
+            "method":"eth_getCode",
+            "params":["INSERT_ADDRESS", "INSERT_BLOCK_VALUE"],
+            "id":1
+        }'
+        ```
+
+        Ensure to replace the `INSERT_ADDRESS` and `INSERT_BLOCK_VALUE` with the proper values.
+
+- [`eth_getStorageAt`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getstorageat){target=\_blank} - returns the value from a storage position at a given address
+
+    - Parameters
+        - Address - contract or account address to query storage
+        - Storage Key - position in storage to retrieve data from
+        - Block Value (optional) - quantity or tag of the [block value](#block-parameter-values) to be fetched
+
+    ???- "Query Example" 
+
+        ```bash title="eth_getStorageAt"
+        curl -X POST https://westend-asset-hub-eth-rpc.polkadot.io \
+        -H "Content-Type: application/json" \
+        --data '{
+            "jsonrpc":"2.0",
+            "method":"eth_getStorageAt",
+            "params":["INSERT_ADDRESS", "INSERT_STORAGE_KEY", "INSERT_BLOCK_VALUE"],
+            "id":1
+        }'
+        ```
+
+        Ensure to replace the `INSERT_ADDRESS`, `INSERT_STORAGE_KEY` and `INSERT_BLOCK_VALUE` with the proper values.
+
+## Block Parameter Values
```

Comment:
I don't think this should be a section by itself. Maybe just a note at the end of the Available Methods section is enough